dsgrid/config/annual_time_dimension_config.py

```diff
@@ -0,0 +1,30 @@
+from datetime import timedelta
+
+from .dimensions import AnnualTimeDimensionModel
+from .time_dimension_base_config import TimeDimensionBaseConfig
+
+
+class AnnualTimeDimensionConfig(TimeDimensionBaseConfig):
+    """Provides an interface to an AnnualTimeDimensionModel."""
```

Comment:
I thought we already had an annual time dimension??? Did it not get merged in earlier?

Reply:
Is this a duplicate of https://github.com/dsgrid/dsgrid/pull/99/files#diff-37f10c1c7dfd83c009a1527d84670f7f08d89e98d8660205c5439e3e99cbef5cL142? (Why are you adding a class with the same name as another class (even if it is in a different file)?)